### DISCUSSION TITLE
[Feature] Accented letters behave as normal letters

### DISCUSSION THREAD

**Discussion by DavidAremaCarretero** [immich-app/immich#13195] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

As a user of people search in Immich
I want to be able to search for my people, which their names have accented letters, with no need to put the accent on the letter 
So that I can filter out people using some letters with no need to differ from accent to no accent.

More context: example, typing "Al", I would like to find those named Alfonso as well as Álvaro. Right now A, Á, À... are different characters (as it is), but are different characters on the ASCII table, not for the language as it's the same letter.

At the end of the day, is to make the search accent insensitive, I'm talking on people as it's the search input I realized this is happening but I guess is something happening on the whole system.

Thanks a lot!!

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by maderapapeonces** [comment]
Was about to open the same suggestion, but I knew somebody had done it before!

Most of well-known applications behave this way: 

- searching for "joao" matches João
- searching for "pogacar" matches Pogačar
- searching for "aguero" matches Agüero

**Comment by headtr1ck** [comment]
Would be really nice to have this.
Even better would be a full fuzzy search to also catch typos.
